plugins/modules/ndo_fabric_span_session.py

```diff
@@ -0,0 +1,705 @@
+    - The UUID of the SPAN Session.
+    - This parameter is required when the O(name) needs to be updated.
+    type: str
+    aliases: [ span_session_uuid ]
+  description:
+    description:
+    - The description of the SPAN Session.
+    type: str
+  admin_state:
+    description:
+    - The administrative state of the SPAN Session.
+    - Defaults to C(enabled) when unset during creation.
+    choices: [ enabled, disabled ]
+    type: str
+  mtu:
+    description:
+    - The MTU truncation size for the SPAN packets.
+    - The value must be in the range 64 - 9216.
+    - Defaults to 1518 when unset during creation.
+    type: int
+  destination_epg:
+    description:
+    - The destination EPG configuration group.
+    - This parameter or O(destination_port) or O(destination_port_channel) is required when creating the SPAN Session.
+    type: dict
+    suboptions:
+      epg_uuid:
+        description:
+        - The UUID of the destination EPG to use for the SPAN Session.
+        - This parameter or O(destination_epg.epg) is required.
+        type: str
+      epg:
+        description:
+        - The destination EPG to use for the SPAN Session.
+        - This parameter or O(destination_epg.epg_uuid) is required.
+        type: dict
+        suboptions:
+          name:
+            description:
+            - The name of the destination EPG.
+            type: str
+            required: true
+          template:
+            description:
+            - The name of the template that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.template_id) is required.
+            type: str
+          template_id:
+            description:
+            - The ID of the template that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.template) is required.
+            type: str
+          schema_id:
+            description:
+            - The ID of the schema that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.schema) is required.
+            type: str
+          schema:
+            description:
+            - The name of the schema that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.schema_id) is required.
+            type: str
+          anp:
+            description:
+            - The name of the ANP that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.anp_uuid) is required.
+            type: str
+          anp_uuid:
+            description:
+            - The UUID of the ANP that contains the destination EPG.
+            - This parameter or O(destination_epg.epg.anp) is required.
+            type: str
+      destination_ip:
+        description:
+        - The destination IP address to route SPAN Session packets.
+        type: str
+      source_ip_prefix:
+        description:
+        - The destination IP address prefix to route SPAN Session packets.
```

Comment:
```suggestion
        - The prefix used to assign source IP addresses to ERSPAN packets which can be used to identify which Leaf or Spine is sending the traffic.
        - This can be any IP. If the prefix is used, the node ID of the source node is used for the undefined bits of the prefix.
```